osd/cluster-wide_proxy_unreachable.json

```diff
@@ -0,0 +1,7 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "summary": "Action required: update cluster proxy state/configuration",
+    "description": "The cluster-wide openshift proxy is unreachable from Openshift network: ${REASON}, generating alerts to Red Hat SRE. Please review proxy server state and configuration to restore the cluster's functionality.,
```

Comment:
```suggestion
    "description": "The cluster-wide proxy configured for your OpenShift cluster is currently reporting as unavailable to the cluster: ${REASON}. This impacts your cluster's network egress, including ability to pull system images, run cluster operators, manage instances and perform upgrade-related actions. Please review your proxy server's state and configuration to restore the cluster's functionality. If you require further assistance, please file a support request.",
```

Reply:
We should also eventually include a reference to the product documentation (when it exists).